### PR TITLE
feat: new component for externals links

### PR DESCRIPTION
**Pull request by DespauxGuillaume**
Adding a component icon for externals links on hover  and refactorisation headerNavigation

### CODE REVIEW

**Comment by DespauxGuillaume** [comment]
@clemcode

**Comment by Atinux** [comment]
I am on phone so cannot check preview but I think on mobile we shall display the external icon (not on hover)

File: components/organisms/app/HeaderNavigation.vue [hunk 1]
```diff
@@ -1,93 +1,55 @@
...
       :blank="blank"
       :class="{
         'text-primary': currentSlug === slug,
         'hover:d-primary-text-hover': currentSlug !== slug
       }"
     >
-      {{ $t(`header.${title}`) }}
+      <div class="flex items-center">
+          {{ title }}
+          <Component v-if="icon" :is="icon" class="w-4 h-4 ml-2 text-gray-400 opacity-0 group-hover:opacity-100"/>
```
Comment (Atinux) [inline components/organisms/app/HeaderNavigation.vue:17]: I guess a md:group-hover

Comment (DespauxGuillaume) [reply components/organisms/app/HeaderNavigation.vue:17]: @Atinux would you mean only for desktop in this case i think is a lg:group-hover ?

Comment (Atinux) [reply components/organisms/app/HeaderNavigation.vue]: Yes

**Comment by DespauxGuillaume** [comment]
@Flosciante

**Review by Flosciante — APPROVED** [review]
👍

**Pull request merged**